package.json

```diff
@@ -76,8 +76,8 @@
     "hoist-non-react-statics": "^1.2.0",
     "path-to-regexp": "^1.7.0",
     "prop-types": "^15.5.8",
-    "react-native-drawer-layout-polyfill": "1.2.0",
-    "react-native-tab-view": "^0.0.61"
+    "react-native-drawer-layout-polyfill": "1.3.0",
+    "react-native-tab-view": "^0.0.62"
```

Comment:
0.0.62 does fix the endless loop issue on iOS but breaks nested tabs again on Android